packages/rancher-sachet/charts/templates/configmap-pre-install.yaml

```diff
@@ -12,6 +12,17 @@ metadata:
     "helm.sh/resource-policy": keep
 data:
   config.yaml: |-
+    # please refer to the upstream documentation for configuration options:
+    #   https://github.com/messagebird/sachet/blob/master/examples/config.yaml
```

Comment:
Should this be pointing to a link on the docs site?

Reply:
Sure. 
I need to confirm with the Doc team to see if we will mention this in Rancher's documentation and  
get the proper URL.

Reply:
Well, it doesn't have to be, I do not believe. I was moreso curious if this was intentional. If so, I think that's fine, but @cbron might have a different take.

Reply:
In this case I'd prefer to leave it pointing at master branch of upstream. As it changes over time I don't want our docs team to have to keep up to date with it. Though I think we should just point at the repo itself instead of the file, in case the file moves.

Reply:
Good point. Agreed